packages/xo-web/src/xo-app/settings/audit/index.js

```diff
@@ -226,20 +239,104 @@ const COLUMNS = [
     sortCriteria: 'time',
     sortOrder: 'desc',
   },
+  {
+    itemRenderer: ({ id }, { checkedRecords, missingRecord }) => {
+      if (missingRecord === undefined && isEmpty(checkedRecords)) {
```

Comment:
Why `isEmpty(checkedRecords)`? Why not simply `checkedRecords[id] === undefined`?